extensions/EXT/EXT_framebuffer_blit_layers.txt

```diff
@@ -0,0 +1,107 @@
+Name
+
+    EXT_framebuffer_blit_layers
+
+Name Strings
+
+    GL_EXT_framebuffer_blit_layers
+
+Contacts
+
+    Simon Zeni, Status Holdings Ltd. <simon@zeni.ca>
+
+Contributors
+
+    Simon Zeni
+
+Status
+
+    Draft
+
+Version
+
+    Version 0.1, 2022/09/16
+
+Number
+
+    OpenGL Extension #558
+    OpenGL ES Extension #341
+
+Dependencies
+
+    OpenGL ES 3.0 is required
+
+    This extension is written against the OpenGL ES 3.2 specification
```

Comment:
The edits below are against the OpenGL ES 3.0 extension, which conflicts with this statement. Do the below edits need to be moved to OpenGL ES 3.2?

Reply:
Also since this extension is for OpenGL too it should say which version of the OpenGL extension it affects.

Reply:
I don't think it needs anything specific from ES 3.2, I'll move it back to ES 3.0.